Makefile

```diff
@@ -100,7 +106,11 @@ tests_functional:
 
 firmware:
 	python3 tools/check_version.py ./config/os_version
-	./tools/firmware/build_firmware.sh -r -v $(FIRMWARE_VERSION)
+	./tools/firmware/build_firmware.sh -r -v $(OS_VERSION)
+
+firmware_no_cleanup:
+	python3 tools/check_version.py ./config/os_version
+	./tools/firmware/build_firmware.sh -v $(OS_VERSION)
```

Comment:
What's this target for?

Reply:
when you're already configured the project to build the firmware and make changes in `os/main.cpp` you don't want to clean up everything and start from scratch each time.

especially when you live in Bretagne and the internet is super slow and it takes ~5/10min to download the dependencies for the project 😂